aimmo-game-creator/worker_manager.py

```diff
@@ -285,6 +308,7 @@ def create_worker(self, id, data):
             else:
                 raise
         self._create_game_rc(id, data)
+        self._add_path_to_ingress(id)
```

Comment:
will we need to remove paths from ingress at some point?

Reply:
When we delete games, but we don't have that functionality yet

Reply:
👍